constantvariables.py

```diff
@@ -6,4 +6,16 @@
 
 VERIFIED_MEMBER_ID = 1120501110407434301 # Replace with the actual ID of the role you want for the greetings message
 
-WELCOME_REACT_EMOJI = "👍" # Replace with the emoji you want to use for your welcome message react emoji
\ No newline at end of file
+WELCOME_REACT_EMOJI = "👍" # Replace with the emoji you want to use for your welcome message react emoji
+
+GAME_CHANNELS = { # Key must be video game name and value must be maximum number of players that can play in a full squad
+    "Valorant": 5,
+    "League of Legeneds": 5,
+    "Rocket League": 3
+}
+
+DIFFERENT_NAME_CHANNELS = { # REQUIRED: Must have lounge, study, and cinema channels. You can name them whatever you want but make sure to replace the channel IDs with your server's
```

Comment:
Can we make a ticket for not requiring these channels? Have a check that if these channels do not exist in the constantvariables.py, then we just skip down the if/elif. We shouldn't have the bot break if servers have different configuration from ours.

Reply:
Must be required if developers want to use different names for their voice channels. For example, "Ichiraku" is known as the "Lounge" channel in our own server. Hard to know if "Ichiraku" is a lounge channel without context. Therefore it is necessary to require those channels to allow flexibility

Reply:
I completely agree with naming and that we can't figure out lounges without context. But the "required" makes it seem like in order to use this bot, you must have lounge, study, and cinema channels. Which means in order to use our bot, your server needs to have a minimum of 3 voice channels. Some servers might want to only have 1-2 channels total and we can't be the ones that determine that. So we need flexibility so that the bot doesn't crash if we do a lookup for a "cinema" voice channel and it doesn't exist.

Reply:
As explained before, we need these different channels if we want to have usage of the bot. If we only have 1-2 channels then the bot does not serve much of a purpose in automove

Reply:
That's exactly the case. We're offering different features with the bot but it's not like we are turning on/off features. It's always good to have checks in case people do something they're not supposed to. 

Say there becomes a case where people take Rasenbot to mainly use it for the time tracking feature @benjaminl4917 is working on. The person who adds our bot takes a look at our Github wiki and sees the automove command. Without thinking they call the command and suddenly their bot crashes. There should never be a case where bot simply crashes. We need to keep the bot running through as many things as possible and output out error logs. 

If you want to make it required for people to have at least 3 channels to use Automove, that's fine. But then we should output an error log to the console for those that have less than 3 channels or don't have the correct specified channels and still attempt to call Automove.